test/remote/gateways/remote_braintree_blue_test.rb

```diff
@@ -15,6 +16,21 @@ def setup
       billing_address: address(country_name: 'Canada'),
       description: 'Store Purchase'
     }
+
+    ach_mandate = 'By clicking ["Checkout"], I authorize Braintree, a service of PayPal, ' \
```

Comment:
Very small thing but let's normalize this to what it would actually be sent as
```suggestion
    ach_mandate = 'By clicking "Checkout", I authorize Braintree, a service of PayPal, ' \
```